include/RAJA/util/View.hpp

```diff
@@ -63,20 +64,24 @@
 namespace RAJA
 {
 
-template <typename DataType,
-          typename LayoutT,
-          typename DataPointer = DataType *>
+template <typename DATA_TYPE,
+          typename LAYOUT_TYPE,
+          typename DATA_POINTER = DATA_TYPE *>
 struct View {
-  LayoutT const layout;
+  using DataType = DATA_TYPE;
+  using DataPointer = DATA_POINTER;
+  using LayoutType = LAYOUT_TYPE;
```

Comment:
That makes the template arguments visible outside of the class.  That's used in the wrapper to grab the DataType.